src/components/ConfigurationStructureDiagram.tsx

```diff
@@ -0,0 +1,128 @@
+import React, {useState} from "react";
+import "@site/src/css/configuration-explorer-layout.css";
+import { Icon } from '@iconify/react';
+
+const folderIcon = "mdi:folder";
+const fileIcon = "mdi:file";
+
+interface ExplorerNode {
+    name: string;
+    type: "folder" | "file";
+    children?: ExplorerNode[];
+    description?: string;
+    url?: string;
+}
+
+const folderData: ExplorerNode[] = [
+    {
+        name: "config",
+        type: "folder",
+        children: [
+            { name: "paper-global.yml", type: "file", url: "/paper/reference/global-configuration" },
+            { name: "paper-world-defaults.yml", type: "file", url: "/paper/reference/world-configuration" },
+        ],
+    },
+    {
+        name: "plugins",
+        type: "folder",
+        description: "You can place your plugin jars here.",
+    },
+    {
+        name: "<world>",
+        type: "folder",
+        children: [
+            { name: "paper-world.yml", type: "file", url: "/paper/reference/configuration#per-world-values", description: "Every world folder will have this file. The values here only apply to this world." },
+        ],
+    },
+    { name: "banned-ips.json", type: "file", description: "This file stores all the banned IP addresses on the server." },
+    { name: "banned-players.json", type: "file", description: "This file stores all the banned player information for the server." },
+    { name: "bukkit.yml", type: "file", url: "/paper/reference/bukkit-configuration" },
+    { name: "commands.yml", type: "file", url: "/paper/reference/bukkit-commands-configuration" },
+    { name: "eula.txt", type: "file", description: "This file is in place to allow you to accept the Minecraft EULA.\nThis is required to start the server." },
+    { name: "help.yml", type: "file", description: "This file provides you with a wide variety of ways to configure the /help system in your Paper Server." },
+    { name: "ops.json", type: "file", description: "ops.json is a JSON file located in the root directory of a server containing a list of players with operator status." },
```

Comment:
```suggestion
    { name: "ops.json", type: "file", description: "This file stores a list of all players with operator status." },
```
Or something like this